### DISCUSSION TITLE
Disable tab pinning feature

### DISCUSSION THREAD

**Discussion by jezikk** [zed-industries/zed#18498] [Feature Requests]
Hi all,

I don't like how to work the whole concept of pinned tabs. Is it possible to turn this feature off? If I want to simply close the tab using X button, I have to firstly unpin the tab and after that it is possible to close the tab.

Thanks,
Michal

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
